tests/testthat/test-translate-package.R

```diff
@@ -69,7 +69,7 @@ test_that("translate_package works on a simple package", {
     {
       expect_messages(
         translate_package(pkg, "zh_CN", verbose=TRUE),
-        c("Beginning new translations", "BEGINNING TRANSLATION", '"Installing" translations with msgfmt'),
+        c("Beginning new translations", "BEGINNING TRANSLATION", "Generating en@quot translations"),
```

Comment:
Windows doesn't generate `en@quot` so this test is failing on the windows build :smiley:

Reply:
Why not? Because `tools:::en_quote()` doesn't work on a non-unicode locale?